ui/packages/shared/profile/src/ProfileIcicleGraph/index.tsx

```diff
@@ -35,6 +35,8 @@ const ProfileIcicleGraph = ({
   const compareMode = useAppSelector(selectCompareMode);
   const {ref, dimensions} = useContainerDimensions();
 
+  console.log('graph', JSON.stringify(graph));
```

Comment:
you probably missed this but there's a stray `console.log` here.

Reply:
Done, taken care of.